File: NewsBrowser/NewsBrowser/ClientApp/src/app/models/SimpleNews.ts
import { Tag } from "./tag";

export interface SimpleNews {
  id: string;
  url: string;
  title: string;
  author: string;
  text: string;
  tags: Tag[];
}

export const EMPTY_SIMPLE_NEWS = {
  id: "",
  url: "",
  title: "",
  author: "",
  text: "",
  tags: []
}

File: NewsBrowser/NewsBrowser/DTOs/SimpleNews.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace NewsBrowser.DTOs
{
    public class SimpleNews
    {
        public string Id { get; set; }
        public string Url { get; set; }
        public string Title { get; set; }
        public string Author { get; set; }
        public string Text { get; set; }
        public List<string> Tags { get; set; }
    }
}
